meteor-app/imports/ui/pages/workspace/component.js

```diff
@@ -149,62 +149,60 @@ export default class WorkspacePage extends React.Component {
         </fieldset>
         <fieldset>
           <legend>Map</legend>
+          <ul className="layer-list">
+            {layers.map((layer, layerIndex) => (
+              <li key={layerIndex}>
+                <div>
+                  <input title="Toggle Visibility" type="checkbox" checked={!layer.invisible} data-layer-index={layerIndex} onChange={this._bound_layerVisibilityOnChange} />
+                  <label>{layer.name}</label>
+                </div>
+                <div>
+                  <label>Opacity: </label>
+                  <input type="range" min="0" max="255" step="1" value={layer.opacity * 255} data-layer-index={layerIndex} onChange={this._bound_layerOpacityOnChange} />
+                  <label>{layer.opacity.toFixed(2)}</label>
+                </div>
+              </li>
+            ))}
+          </ul>
           <div className="section_map">
-            <ul className="layer-list">
```

Comment:
Depending on the merge order, this may or may not cause a conflict. My bet is on it will. Also unnecessary changes are unnecessary changes. You don't add a piece of irrelevant change in a PR and say "I'm going to use it later for another thing". Just don't do that.